src/EditorFeatures/VisualBasic/TextStructureNavigation/TextStructureNavigatorProvider.vb

```diff
@@ -41,19 +38,31 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.TextStructureNavigation
                     Return True
 
                 Case SyntaxKind.CharacterLiteralToken
-
-                    ' Before the ' is considered outside the character
+                    ' Before the opening quote is considered outside the character
                     If position = token.SpanStart Then
                         Return False
                     End If
 
                     Return True
 
-                Case SyntaxKind.XmlTextLiteralToken
+                Case SyntaxKind.InterpolatedStringTextToken,
+                     SyntaxKind.XmlTextLiteralToken
                     Return True
             End Select
 
             Return False
         End Function
+
+        Protected Overrides Function GetExtentOfWordFromToken(token As SyntaxToken, position As SnapshotPoint) As TextExtent
+            If token.Kind() = SyntaxKind.StringLiteralToken AndAlso position.Position = token.Span.End - 1 Then
+                ' Special case to treat the closing quote of a string literal as a separate token.  This allows the
+                ' cursor to stop during word navigation (Ctrl+LeftArrow, etc.) immediately before AND after the
+                ' closing quote, just like it did in VS2013 and like it currently does for interpolated strings.
+                Dim Span = New Span(position.Position, 1)
+                Return New TextExtent(New SnapshotSpan(position.Snapshot, Span), isSignificant:=True)
+            Else
+                Return MyBase.GetExtentOfWordFromToken(token, position)
```

Comment:
Since `(C#)SyntaxKind.StringLiteralToken != (VB)SyntaxKind.StringLiteralToken` I didn't see a clean way of doing this so I opted to keep them separate, but I could certainly be talked either way.